buildroot-v86/board/v86/linux.config

```diff
@@ -156,47 +157,48 @@ CONFIG_NVRAM=y
 CONFIG_HPET=y
 # CONFIG_HPET_MMAP is not set
 CONFIG_I2C_I801=y
-CONFIG_PTP_1588_CLOCK=y
-CONFIG_WATCHDOG=y
-CONFIG_AGP=y
-CONFIG_AGP_AMD64=y
-CONFIG_AGP_INTEL=y
-CONFIG_DRM=y
-CONFIG_DRM_I915=y
-CONFIG_FB_MODE_HELPERS=y
-CONFIG_FB_TILEBLITTING=y
-CONFIG_FB_EFI=y
+# Graphics drivers is not set
```

Comment:
Every one of these begins `CONFIG_` so a key like `Graphics drivers` doesn't make sense to me.  Is this just a comment?

Reply:
Yes, in reference to all of the graphics drivers

Reply:
Remove this line, it's confusing.

Also, can you please report on the build size of these changes as I asked previously?  Thanks.

Reply:
Sorry, I was waiting to build it on my machine. It came down to 4.2 mb's.